python_bindings/tests/engine_test.py

```diff
@@ -85,3 +91,102 @@ def test_game_engine_send_batch_returns_failures_when_communicator_closed(
         expected = requests[idx].GameID
         actual = resp.GameID()
         assert actual == expected
+
+
+def test_game_engine_send_get_remaining_tiles_batch_returns_remaining_tiles(
+    tmp_path: Path
+) -> None:
+    t1 = tiletemplates.MonasteryWithSingleRoad()
+    t2 = tiletemplates.RoadsTurn()
+    tiles = [t1, t2, t1]
+    tile_set = TileSet(
+        StartingTile=tiletemplates.SingleCityEdgeStraightRoads(),
+        Tiles=Slice_tiles_Tile(tiles),
+    )
+
+    engine = StartGameEngine(1, str(tmp_path))
+    game = engine.GenerateGame(tile_set)
+
+    request = GetRemainingTilesRequest(BaseGameID=game.ID)
+    resp, = engine.SendGetRemainingTilesBatch(
+        Slice_Ptr_engine_GetRemainingTilesRequest([request])
```

Comment:
Some wrapper methods would be nice to avoid such explicit casting

Reply:
It's an annoyance with auto-generated bindings. There's no way to get gopy to generate methods that would do the casting on their own though I'm pretty sure gopy meant for it to just work and is just one of its bugs.
I should be able to write wrapper functions and assign them to the class's namespace to become methods since Python allows for that but it's going to be manual and slightly hacky.

Reply:
> I should be able to write wrapper functions and assign them to the class's namespace to become methods
This would allow us to become independent from auto-generated bindings. And also would yield a more stable API.